### PR TITLE
feat(behavior_velocity_planner): output missing inputs

### PR DESCRIPTION
**Pull request by takayuki5168**
Signed-off-by: Takayuki Murooka <takayuki5168@gmail.com>

## Description

Currently, it's hard to see the topic which is missing for the initialization of behavior velocity planner.
Therefore, I added the output of the missing topic.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: planning/behavior_velocity_planner/src/node.cpp [hunk 1]
```diff
@@ -203,35 +203,46 @@ BehaviorVelocityPlannerNode::BehaviorVelocityPlannerNode(const rclcpp::NodeOptio
...
 // NOTE: argument planner_data must not be referenced for multithreading
-bool BehaviorVelocityPlannerNode::isDataReady(const PlannerData planner_data) const
+bool BehaviorVelocityPlannerNode::isDataReady(
+  const PlannerData planner_data, rclcpp::Clock clock) const
 {
   const auto & d = planner_data;
 
   // from tf
   if (d.current_pose.header.frame_id == "") {
+    RCLCPP_INFO(get_logger(), "Frame id of current pose is missing");
```
Comment (taikitanaka3) [inline planning/behavior_velocity_planner/src/node.cpp:213]: @takayuki5168 
Can you also change this to throttle? to find out missing topic easier

Comment (taikitanaka3) [reply planning/behavior_velocity_planner/src/node.cpp:213]: @takayuki5168 
ping

Comment (takayuki5168) [reply planning/behavior_velocity_planner/src/node.cpp:213]: pong

Comment (takayuki5168) [reply planning/behavior_velocity_planner/src/node.cpp]: https://github.com/autowarefoundation/autoware.universe/pull/1477/commits/af6157f9e7d449828f725ab840a8f461b16c844f
DONE

**Review by taikitanaka3 — APPROVED** [review]
LGTM

**Pull request merged**